src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -23,10 +28,13 @@ public IntakeSubsystem() {
   }
 
   public void toggleIntake(boolean reverse) {
+    ShuffleboardTab OperatorView = Shuffleboard.getTab("Intake Speed");
+    OperatorView.add("Intake Wheel Speed", Constants.intakeMotorSpeed).withWidget(BuiltInWidgets.kDial);
```

Comment:
Should we care whether intake is running forward or reverse for this display (positive or negative value)? Since it is a constant value and we're not providing a way to overwrite it, I'm not sure this is helpful on its own

Reply:
@Majorferguson21 had asked for this to be a value instead of a boolean or string.  Thoughts, Zane?

Reply:
We should probably make sure this is a NetworkTableEntry, so it can be updated as the direction of the intake changes.
A value is best, so we can show the direction the intake is spinning on a slider-type widget using only 1 entry and only 1 widget.

Reply:
Looking again, I see what you mean Siana. We should putNumber after line 36 with -Constants.intakeMotorSpeed, and after line 41 with Constants.intakeMotorSpeed.  That will show the changes when the motor is reversed.

Also, let's put the "add" in the constructor of the subsystem and put a putNumber call here.

Reply:
This has already been addressed in the latest version of main, as this PR is quite old.